tests/test_readme.py

```diff
@@ -64,6 +64,15 @@ def test_chat_with_model():
     assert "What food do llamas eat?" in result.stdout
 
 
+@mock.patch.dict(os.environ, {"LT_ACCELERATOR": "gpu"})
```

Comment:
BNB supports only CUDA.
You need to add RunIf statement.

Reply:
Hm weird it worked for me locally like that without